mlip3_missing_files/mlp_commands.cpp

```diff
@@ -0,0 +1,2192 @@
+/*   MLIP is a software for Machine Learning Interatomic Potentials
```

Comment:
Did. you change something in this code compared to the vanilla version in their repo? If so, please explain
the changes you did in an added section of the header.

Reply:
a call to a function was commented out. I added a header to explain.